Initalized Gameboard

HectorDelgado opened the pull request:
Initialized empty 5x5 game board

HectorDelgado committed:
Initializing Game board

Creating game board

HectorDelgado committed:
Merge remote-tracking branch 'MarcoASedano/master' into HectorsTest

# Conflicts:
#	BattleshipGame.java

HectorDelgado committed:
Created Empty Game Board

Initialized empty 5x5 game board

MarcoASedano commented:
Let's try not to add the .class files, only the java files. Also, lets try to get our model down first before we start implementing any of the logic/gameplay.

I think the Gameboard should be it's own class (since each player will have two, and the board will be interacting with players and ships).

Try to abstract it out before we start adding anything to the Battleshipgame .

HectorDelgado committed:
Merge remote-tracking branch 'MarcoASedano/master' into HectorsTest

HectorDelgado committed:
Initialized methods

Removed my gameBoard code from BattleshipGame.java. Initialized
adjustedColumnNumber & adjustedRowNumber. Added Note at Line 19

MarcoASedano commented on Gameboard.java:
```diff
@@ -15,6 +15,8 @@
  	private static FINAL int ROWS = 10;
  	private static FINAL int COLUMNS = 10;
+     
+    //isn't it [ROWS][COLUMNS] for a two-dimensional array?
```
Yes, it usually is row columns. However, for battleship specifically the columns are letters and rows are numbers, and when you chose a spot to attack, you always say A-4 or B-10 (so column first, then row)

MarcoASedano commented on Gameboard.java:
```diff
@@ -43,10 +45,32 @@ else if (row > 10 || row < 1) {
- }
\ No newline at end of file
+    }//end method adjustedRowNumber
+ }//end class Gameboard
```
So lets try to have our brackets/indentation all be the same for all of our files...so fix your indentations and put the first open bracket in the same line as the method name.

the adjustedRowNumber method implementation looks correct.

The adjustcolumnNumber method is a little off I think...First of all, i messed up. the parameter should be a char type, not a string (since we'll only ever have a-j)
All that method is supposed to do is take a char and return an int. I think the easier way to implement this is with switch statements. look them up and let me know if you have trouble implementing it.

MarcoASedano commented:
Look at my comments on the Gameboard.java class

HectorDelgado committed:
Merge remote-tracking branch 'MarcoASedano/master' into HectorsTest

# Conflicts:
#	Gameboard.java

HectorDelgado committed:
Updated adjustColumnNumber

Changed adjustColumnNumber to switch statement.

MarcoASedano commented:
The method implementation looks good!
